### ISSUE TITLE
Does someone use the wav2vec2  in production?

### ISSUE THREAD

**Issue by Denovitz** [pytorch/fairseq#3023]
If someone uses please share your experience

**Comment by YUYUNHUANG** [comment]
I tried, the issue is that with Tx LM, it is very very very slow, the accuracy is pretty good and comparable to these vendors even without finetune.

**Comment by inboxpraveen** [comment]
I use it but it takes a hell lot of memory. Still not able to figure out how to reduce memory usage. Any pointers anyone?
